Documentation/CRDs/Object-Storage/ceph-object-realm-crd.md

```diff
@@ -17,6 +17,7 @@ metadata:
 spec:
   pull:
     endpoint: http://10.2.105.133:80
+  defaultRealm: true
```

Comment:
Should we also add this example to the `object-multi*.yaml` examples? Or just this doc, to avoid the users specifying defaultRealm when it was not intended?

Reply:
What if you add it to the examples, but leave it as false in those yamls? This way the setting will be more discoverable, but not run the risk of accidentally enabling a default realm.